core/common/src/test/java/alluxio/CommonTestUtils.java

```diff
@@ -190,4 +199,57 @@ public static void waitFor(String description, Function<Void, Boolean> condition
     }
     return fields;
   }
+
+  /**
+   * @return the directory to be used for temporary files in Alluxio tests
+   */
+  public static synchronized File getAlluxioTestDirectory() {
```

Comment:
Is this intentionally `public`? Can the variable be accessed directly? Nothing external calls this...